pkg/translator/loki/encode_test.go

```diff
@@ -27,21 +27,21 @@ func exampleLog() (plog.LogRecord, pcommon.Resource) {
 	buffer := plog.NewLogRecord()
 	buffer.Body().SetStringVal("Example log")
 	buffer.SetSeverityText("error")
-	buffer.Attributes().PutString("attr1", "1")
-	buffer.Attributes().PutString("attr2", "2")
+	buffer.Attributes().UpsertString("attr1", "1")
```

Comment:
`UpsertString` is deprecated in 0.60.0, `PutString` was the right method

Reply:
Indeed, I got warnings before, but that was before using v0.60.0. Thanks!